Fibo1.js

```diff
@@ -10,4 +10,4 @@ function fibonaci(n) {
     }
 }
 
-console.log('num is', fibonaci(20));
\ No newline at end of file
+console.log('num is sambosak', fibonaci(20));
```

Comment:
This doesn't make any sense :(
Also it conflicts with other PRs

Reply:
we use the loacker schema here! not samboosah